src/main/java/org/folio/circulation/domain/policy/OverdueFinePolicy.java

```diff
@@ -2,9 +2,49 @@
 
 import io.vertx.core.json.JsonObject;
 
+import java.util.HashMap;
+import java.util.Map;
+
 import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
 
 public class OverdueFinePolicy extends Policy {
+  public enum OverdueFineInterval {
```

Comment:
If the intent is for this to be a nested type, I think it should be at the bottom of the file rather than at the top

Reply:
Moved to the bottom.